discord/abc.py

```diff
@@ -1197,6 +1201,8 @@ async def send(self, content=None, *, tts=False, embed=None, file=None,
             Indicates if the message should be sent using text-to-speech.
         embed: :class:`~discord.Embed`
             The rich embed for the content.
+        embeds: List[:class:`~discord.Embed`]
+            A list of embeds to upload. Must be a maximum of 10.
```

Comment:
I think is better to state that is added in version 2.0, like view.

Reply:
```suggestion
        embeds: List[:class:`~discord.Embed`]
            A list of embeds to upload. Must be a maximum of 10.
            
            .. versionadded:: 2.0
```